README.md

```diff
@@ -1,25 +1,26 @@
-# Installer for qbittorrent (Dark version)
+# Installer for qbittorrent (Dark Theme)
```

Comment:
It is not just dark theme, it is fork with dark interface included.

Reply:
"Theme" seems more grammatically correct in this use case, as theme implies that it changes the interface, in this case to dark.

Excerpt from google:
> A theme is a preset package containing graphical appearance details.

Reply:
But this installer is for the whole qBittorrent(https://github.com/suratovvlad/qBittorrent) with dark theme included, not just theme.
Anyway, I'm not insisting for this, it should be @suratovvlad decision